cinema/serializers.py

```diff
@@ -68,13 +78,58 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available",
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("id", "row", "seat")
```

Comment:
we'll have redundant movie session id in each item of the "taken_places" when we get one movie session - see the example in the task